packages/systemd/9004-core-add-separate-timeout-for-system-shutdown.patch

```diff
@@ -0,0 +1,66 @@
+From 4d11f5d502ca4a61c491681cdfd99ebe24e3f58c Mon Sep 17 00:00:00 2001
+From: Ben Cressey <bcressey@amazon.com>
+Date: Tue, 10 Mar 2020 20:30:10 +0000
+Subject: [PATCH 9004/9004] core: add separate timeout for system shutdown
```

Comment:
Is this something we'd want to contribute upstream?

Reply:
Perhaps, though it's less urgent than removing the need for the other three patches.

Reply:
Do we have an issue tracking the removal or upstreaming of these patches?

Reply:
For the other patches, removal is implied by #248, #249, #250.

For this patch, I'm not super clear on the customer value that we'd be delivering by upstreaming the patch - even assuming it's acceptable upstream - which makes it hard to know what the issue should say.

I'm generally OK with carrying a limited number of small local patches that are specific to our distro since that's less likely to cause an ongoing maintenance burden. I could add a "TODO: rework this for upstream" to the spec file if that seems useful.

Reply:
I think the description in the patch covers why it's useful, at least to a subset of distros.  A TODO would be helpful, if nothing else.

Reply:
I've added the TODO note.